### PR TITLE
Stop integration test from sending real email via AmazonSES MAILPOET-4394

### PR DESCRIPTION
**Pull request by samnajian**
Currently .env variables are not picked up for integration tests, this PR enables this.

[MAILPOET-4394]

[MAILPOET-4394]: https://mailpoet.atlassian.net/browse/MAILPOET-4394?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by lysyjan — CHANGES_REQUESTED** [review]
This pull request doesn't solve the issue. We need to continue with the discussion on Slack and decide how to deal with Amazon sending in integration tests.
Also, we should restore the removed sending test.

**Comment by samnajian** [comment]
> This pull request doesn't solve the issue. We need to continue with the discussion on Slack and decide how to deal with Amazon sending in integration tests. Also, we should restore the removed sending test.

@lysyjan thanks for taking a look. As explained in the PR description this PR only enables the picking up the .env variables which is what's needed to be able to reproduce the issue documented in MAILPOET-4394 locally.

**Comment by lysyjan** [comment]
@samnajian In my opinion, this pull request should also solve the reactivation and repair of the test. If not, I would want to know the next steps because you are assigned to the ticket in the Jira. When we move this pull request further, the test may stay inactive because the discussion in the slack has been almost two weeks without further response.

File: mailpoet/tests/integration/Mailer/Methods/AmazonSESTest.php [hunk 1]
```diff
@@ -255,4 +258,33 @@ public function testItChecksBlacklistBeforeSending() {
     expect($result['error'])->isInstanceOf(MailerError::class);
     expect($result['error']->getMessage())->stringContainsString('AmazonSES has returned an unknown error.');
   }
+
+  public function testItCanSend() {
+    if (getenv('WP_TEST_MAILER_ENABLE_SENDING') !== 'true') $this->markTestSkipped();
```
Comment (lysyjan) [inline mailpoet/tests/integration/Mailer/Methods/AmazonSESTest.php]: I think that we don't need to skip this test anymore when the sending is mocked. What do you think?

**Review by lysyjan — COMMENTED** [review]
It looks good. I have a question about skipping only.

File: mailpoet/tests/integration/Mailer/Methods/AmazonSESTest.php (hunk 1 above)
Comment (samnajian) [reply mailpoet/tests/integration/Mailer/Methods/AmazonSESTest.php]: @lysyjan umm, yea makes sense, updating.

**Pull request merged**